src/api/docs/content/index.html

```diff
@@ -10,14 +10,7 @@
         <script type='text/javascript' src='external/rapidoc-min.js'></script>
         <script type="text/javascript" src="pi-hole.js"></script>
         <link href='index.css' rel='stylesheet'>
-        <link rel="apple-touch-icon" href="/admin/img/favicons/apple-touch-icon.png" sizes="180x180">
-        <link rel="icon" href="/admin/img/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
-        <link rel="icon" href="/admin/img/favicons/favicon-16x16.png" sizes="16x16" type="image/png">
-        <link rel="manifest" href="/admin/img/favicons/manifest.json">
-        <link rel="mask-icon" href="/admin/img/favicons/safari-pinned-tab.svg" color="#367fa9">
-        <link rel="shortcut icon" href="/admin/img/favicons/favicon.ico">
-        <meta name="msapplication-TileColor" content="#367fa9">
-        <meta name="msapplication-TileImage" content="/admin/img/favicons/mstile-150x150.png">
+        <link rel="shortcut icon" href="img/favicon.ico">
```

Comment:
Are you sure the icon will work for every browser if we remove the other lines?

I added the other tags to match what we use for the web interface: https://github.com/pi-hole/web/blob/1eaddca8db2e54a57e682633ea31096a3166b082/scripts/lua/header.lp#L58-L66

Reply:
It is only relevant for the API documentation. If the favicon doesn't work here it is not really a loss...

Reply:
`shortcut icon` is enough from IE and onward. The other two are Windows specific, but I doubt there's anyone using them and in fact we should drop them from the Web repo too...